packages/design-tokens/docs/pages/how-to-use-with-sass.stories.mdx

```diff
@@ -0,0 +1,57 @@
+import { Meta } from '@storybook/addon-docs/blocks';
+import exampleSass from '!!raw-loader!./examples/example.scss'
+import { CodeBlock, LinkToStory } from '../DocsComponents'
+import apiPage from './api.stories.mdx'
+import gettingStartedPage from './how-to-setup.stories.mdx'
+
+<Meta title="How-to guides/How to use design tokens in SASS code" parameters={{ docsOnly: true }} />
+
+# How to use design tokens in SASS code
+
+
+## 1. Setup a ThemeManager
+
+Ensure you have followed the steps in <LinkToStory storyModule={gettingStartedPage} /> to setup your application for using design tokens.
+
+## 2. Start using design tokens
+
+Tokens (which contain CSS variables, e.g `var(--kz-var-color-ash)`) can be subscribed to in SASS by importing variables exposed by the package.
+
+Example:
+
+<CodeBlock language="scss" code={exampleSass} />
+
+See <LinkToStory storyModule={apiPage} hash="sass-variables">SASS Variables</LinkToStory> for all available tokens.
+
+
+## 3. Be aware of the limitations of CSS variables in SASS
+
+### Compile-time functions like `mix()` or `darken()`
+
+Because the exposes SASS variables contain values such as `var(--kz-var-color-wisteria-800)`, CSS compilers such as SASS won't be able to calculate anything that relies on concrete values such as HEX or RGB strings.
+
+**It won't be possible to do things like:**
+<CodeBlock language="scss" code={`.myClass { 
+    color: mix(white, $kz-var-color-wisteria-800, 90%);
+}`} />
+
+because, `$kz-var-color-wisteria-800` will hold a value like `var(--kz-var-color-wisteria-800)`, and SASS won't be able to resolve it to a concrete color value to compute the color blend.
```

Comment:
```suggestion
because, `$kz-var-color-wisteria-800` will hold a value like `var(--kz-var-color-wisteria-800)`, and SASS won't be able to resolve it to a concrete color value to compute the color blend.

Usually when we've used functions like `mix` or `shade` it has been to lighten or darken a token by a certain amount. We recommend you just find the nearest plain value. In the example above,  you could simply use `$kz-var-color-wisteria-600`
```